extensions/data-plane/data-plane-api/src/main/java/org/eclipse/dataspaceconnector/dataplane/api/transfer/DataPlanePublicApiRequestFilter.java

```diff
@@ -90,36 +89,52 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
             return;
         }
 
-        // create request and perform validation
-        var dataFlowRequest = createDataFlowRequest(tokenValidationResult.getContent(), requestContext);
-        var validationResult = dataPlaneManager.validate(dataFlowRequest);
-        if (validationResult.failed()) {
-            requestContext.abortWith(validationResult.getFailureMessages().isEmpty() ?
-                    validationError(String.format("Failed to validate request with id: %s", dataFlowRequest.getId())) :
-                    validationErrors(validationResult.getFailureMessages()));
-            return;
+        // create the data flow request and map it into the request body
+        var body = new BufferedReader(new InputStreamReader(requestContext.getEntityStream()))
```

Comment:
As github suggest, better close the Reader after extracting data, maybe with a `try` with resources statement